kore/src/Kore/Step/RulePattern.hs

```diff
@@ -303,8 +304,16 @@ isNormalRule RulePattern { attributes } =
         Attribute.Normal -> True
         _ -> False
 
-getPriority :: RulePattern variable -> Attribute.Priority
-getPriority = Attribute.priority . attributes
+getPriorityOfRule :: RulePattern variable -> Integer
+getPriorityOfRule RulePattern { attributes } =
+    if isOwise
```

Comment:
Actually, I do have two requests:

1. Instead of hard-coding the values here, add top-level definitions `owisePriority, defaultPriority :: Integer`, and
1. Make `defaultPriority = 50`, see here: https://runtimeverification.slack.com/archives/C7E701MFG/p1582663748043400